### PR TITLE
DOCSUP 3744: Update the SQL-statement CREATE TABLE

### PR DESCRIPTION
**Pull request by antarctictardigrade**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #15823.

### CODE REVIEW

File: docs/en/sql-reference/statements/create/table.md [hunk 1]
```diff
@@ -99,6 +99,90 @@ It is not possible to set default values for elements in nested data structures.
...
+
+``` sql
+CREATE TABLE [IF NOT EXISTS] [db.]table_name (name1 [type1], PRIMARY KEY(name1)) ENGINE = engine ORDER BY (name1);
+```
+
+``` sql
+CREATE TABLE [IF NOT EXISTS] [db.]table_name (name1 [type1]) ENGINE = engine ORDER BY (name1) PRIMARY KEY(name1);
+```
+
+Both of these syntaxes are valid when creating a primary key with only one field.
```
Comment (alexey-milovidov) [inline docs/en/sql-reference/statements/create/table.md:114]: It also works with several fields.

File: docs/en/sql-reference/statements/create/table.md [hunk 2]
```diff
@@ -99,6 +99,90 @@ It is not possible to set default values for elements in nested data structures.
...
+    `a` Int16
+)
+ENGINE = MergeTree
+PRIMARY KEY a
+ORDER BY a
+SETTINGS index_granularity = 8192 │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+You can also define a `PRIMARY KEY` constraint on multiple columns.
```
Comment (alexey-milovidov) [inline docs/en/sql-reference/statements/create/table.md:156]: It's not a constraint.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

**Comment by BayoNet** [comment]
I think we need just close this PR.
Another person will do this task.

**Pull request closed without merging**